### DISCUSSION TITLE
fork into a more active repo?

### DISCUSSION THREAD

**Discussion by JustinGuese** [kernc/backtesting.py#1106] [Polls]
sadly it seems abandoned, but I see a lot of PRs as well, and think it has a nice potential if we would add some more features... so what do you think about forking it into a new open source project where we have more admins? @kernc if you read this I'd be open to your opinion ;)
An alternative would be if kernc adds more people to it such we can manage PRs

Everyone that would be willing to add code please write a comment in the docs ;)

**Comment by msloco** [comment]
whichever is good for me, the point is keeping it alive and evolving - I am not so much of a potential contributor, but for sure an heavy user....

**Reply by BradKML** [reply]
Need more hands on deck to update a lot of the stuff then... anything that allows this to grow is a good idea

**Comment by kernc** [comment]
Fwiw, this repo will be brought up to speed shortly. Without pointing any hands, the remaining and particularly disconcerting issue is the apparent dormancy of the excellent but defunct [scikit-optimize](https://github.com/scikit-optimize/scikit-optimize) project, a soft dependency of ours. In particular, I'm not yet wholly decided how, if, to replace it ...

**Reply by jfncs** [reply]
A different interface but well maintained & documented ; https://github.com/uqfoundation/mystic

**Reply by BradKML** [reply]
Thanks for the note man, would like to get my hands on this and play with it. Also a competitor of VectorBT is always appreciated

**Comment by tsunamilx** [comment]
Inactivity is always the biggest concern for me for any repos no matter how popular it was. I am not expecting some big features, but some small improvements/bug fixes can already be very helpful. I see some PRs are small changes and are actually helpful, any chance to get those PR reviewed?
